File: Laboratory 10/Lab10/RecipeManager-BE/Model/Recipe.cs
namespace RecipeManager.Model
{
    public class Recipe
    {


        public Recipe(int AuthorId, int TypeId, string Name, string Description)
        {
            this.AuthorId = AuthorId;
            this.TypeId = TypeId;
            this.Name = Name;
            this.Description = Description;
            
        }

        public int Id { get; set; }
        public int AuthorId { get; set; }

        public int TypeId { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public Author author { get; set; }

        public Type type { get; set; }
           
    }
}

File: Laboratory 10/Lab10/RecipeManager-FE-Angular/src/app/recipe.ts
import {Author} from "./author";
import {Type} from "./type";

export class Recipe{
  constructor(
    public id: number,
    //public authorId: number,
    //public typeId: number,
    public name: string,
    public description: string,
    public firstName: string,
    public lastName: string,
    public typeDescr: string,

  ){}
}
